widget/embedded/src/pages/Home.tsx

```diff
@@ -203,9 +203,7 @@ export function Home() {
               loadingBalance={fetchingBalance}
               tooltipContainer={getContainer()}
               onSelectMaxBalance={() => {
-                if (fromTokenFormattedBalance !== '0') {
-                  setInputAmount(tokenBalanceReal.split(',').join(''));
-                }
+                setInputAmount(tokenBalanceReal.split(',').join(''));
```

Comment:
it has inconsistent behavior. 
1. Don't select anything (open: `http://localhost:3002/`) then click on Max it will put `0` for input. 
2. Select a token that you have some in your wallet then click on Max it will set the amount that you have. Then select a token that you don't have any, click on Max, it will set '' for input.

Reply:
Answered by the order:
1. I don't get how that inconsistent behavior is related to the related issue or my changes. It is declared in the related issue that "max button doesnt work when balance is 0 (it should work anyway)". So I should make sure that the max button should work even if the balance is equal to zero. I think that the problem you are referring to is about if we should display the max button and the balance if the user did not select any token.
2. That happens because 'fromTokenBalance.balance' is 'undefined' in that situation and the value of 'tokenBalanceReal' will be equal to "". I fixed it by setting the input amount equal to "0" in that situation by changing that line of code to "setInputAmount(tokenBalanceReal.split(',').join('') || '0');"

Reply:
the origin of this bug is `tokenBalanceReal` because `numberToString` is returning `''`. You can fix that line by changing it to something like this:
```
  const tokenBalanceReal =
    !!fromBlockchain && !!fromToken && fromTokenBalance?.amount
      ? numberToString(fromTokenBalance?.amount, fromTokenBalance?.decimals)
      : '0';
```

Reply:
Done